ui/app/pages/routes/routes.component.js

```diff
@@ -98,11 +100,9 @@ export default class Routes extends Component {
     this.props.history.listen((locationObj, action) => {
       if (action === 'PUSH') {
         pageChanged(locationObj.pathname)
-        const url = `&url=${encodeURIComponent('http://www.metamask.io/metametrics' + locationObj.pathname)}`
         this.context.metricsEvent({}, {
-          currentPath: '',
+          currentPath: `${METAMETRICS_TRACKING_URL}${locationObj.pathname}`,
```

Comment:
Isn't this missing a `/`?
```suggestion
          currentPath: `${METAMETRICS_TRACKING_URL}/${locationObj.pathname}`,
```

Reply:
I believe that `pathname` should always start with a leading slash, unless it's empty.

Reply:
You are correct!